buscar-matriz.rb

```diff
@@ -0,0 +1,24 @@
+def buscar(pMatriz, pOtro)  
+_array = []
+_flag = 0
+
+  pMatriz.each do |_opjeto|    
+      if (_opjeto.index{|v| v == pOtro} == nil)
```

Comment:
Its very important create variables more readable also if we have inside a block avoid use this i, v, instead this index, value and so on.

```suggestion
      if (_opjeto.index{ |value| value == pOtro} == nil)
```